src/praisonai/praisonai/cli/app.py

```
@@ -13,6 +13,32 @@
+        set_default_emitter(emitter)
+        
+        # Set up context-level trace emitter
+        ctx_emitter = ContextTraceEmitter(sink=sink, enabled=True)
+        set_context_emitter(ctx_emitter)
+        
+    except ImportError:
+        # Gracefully degrade if Langfuse not installed
+        pass
+    except Exception:
+        # Silently fail to avoid breaking CLI if observability setup fails
+        pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)